src/extract.rs

```diff
@@ -77,9 +77,7 @@ fn extract_links_from_html(input: &str) -> Vec<String> {
 fn walk_html_links(mut urls: &mut Vec<String>, node: &Handle) {
     match node.data {
         NodeData::Text { ref contents } => {
-            // escape_default turns tab characters into "\t", newlines into "\n", etc.
-            let esc_contents = contents.borrow().escape_default().to_string();
```

Comment:
@pawroman, was there a reason why we call `escape_default` here? Removed them because they cause problems with links at the end of a line.
If we need the calls for some other purpose, we can add them back.

Reply:
I think this was to avoid having un-escaped newline, tab, etc. characters in URLs.

To be fair, I recall seeing it in html5ever [example code](https://github.com/servo/html5ever/search?q=escape_default) and just assuming it was the reasonable thing to do.

I guess if no tests are failing then we can remove this, unless a reasonable counter-argument could be found. I assume that the newline and tab characters will be url-escaped anyway.